src/applications/disability-benefits/996/containers/ConfirmationPage.jsx

```diff
@@ -23,23 +23,22 @@ export class ConfirmationPage extends React.Component {
   render() {
     const { submission, data } = this.props.form;
     const { response } = submission;
-    const name = data.veteranFullName;
+    const name = data.fullName;
 
     return (
       <div>
         <h3 className="confirmation-page-title">Claim received</h3>
         <p>
-          We usually process claims within
-          <strong>a week</strong>.
+          We usually process claims within <strong>a week</strong>.
         </p>
         <p>
           We may contact you for more information or documents.
           <br />
           <i>Please print this page for your records.</i>
         </p>
         <div className="inset" role="presentation">
-          <h4>
-            Higher-Level Review Claim
+          <h4 style={{ marginTop: 0 }}>
```

Comment:
This should use the [design system class](https://design.va.gov/utilities/margins#margin-top) `vads-u-margin-top--0`.